### DISCUSSION TITLE
Events processing in dev branch 0.5

### DISCUSSION THREAD

**Discussion by GiODi20** [librespot-org/librespot#1081] [General]
Is there any new documentation on how to process events?

I've just upgraded from 0.2 to 0.5 after a hardware failure and my --onevent script no longer works. I've seen the posts about the events overhaul as well as the example python script, but I just cannot  figure out how to use any of it. Any help would be greatly appreciated

**Comment by roderickvd** [comment]
Should be on the wiki?
